packages/grafana-ui/src/components/PanelMenu/PanelMenu.tsx

```diff
@@ -0,0 +1,302 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { GrafanaTheme2, PanelMenuItem } from '@grafana/data';
+import { css, cx } from '@emotion/css';
+import { useStyles2 } from '../../themes';
+import { PanelMenuListItem } from './PanelMenuItem';
+import { Icon } from '../Icon/Icon';
+import { ClickOutsideWrapper } from '../ClickOutsideWrapper/ClickOutsideWrapper';
+
+interface Props {
+  items: PanelMenuItem[];
+  title: string;
+  buttonClassName?: string;
+  outside?: React.ReactElement;
```

Comment:
Not terribly happy with the names "outside" and "inside" here. Need to come up with something more descriptive.